pkg/cmd/server/api/types.go

```diff
@@ -646,3 +646,75 @@ type PodManifestConfig struct {
 	// The interval needs to be a positive value
 	FileCheckIntervalSeconds int64
 }
+
+type LDAPSyncConfig struct {
+	// URL is a partial RFC 2255 URL which specifices the scheme, host and port of the LDAP server
+	// The syntax of the URL is: `scheme://host:port/`, where `scheme` is either ldap or ldaps
+	URL string
```

Comment:
Looks like you're missing a couple options from `LDAPPasswordIdentityProvider` for making the connection.